python/geocode.py

```diff
@@ -84,8 +85,19 @@ def parse_output(geo):
             distinct ogc_fid as uid, 
             housenumber as house_number,
             streetname as street_name, 
-            borough
-        FROM dob_jobapplications
+            borough,
+            'bis' as source
+        FROM dob_jobapplications UNION
+        SELECT 
+            distinct ogc_fid as uid, 
+            regexp_replace(
+                trim(house_no), 
+                '(^|)0*', '', ''
+            ) as house_number,
+            trim(street_name) as street_name, 
```

Comment:
Why does this only apply to NOW, and not BIS?

Reply:
not sure, because we didn't have this step in the bis init step?

Reply:
I can also apply it to both

Reply:
The leading zero cleaning came from BIS, not from NOW.

Reply:
ahhh I see